test/ae_mdw_web/controllers/tx_controller_test.exs

```diff
@@ -25,4 +30,36 @@ defmodule AeMdwWeb.TxControllerTest do
       assert %{"error" => _error_msg} = conn |> get("/tx/#{tx_hash}") |> json_response(404)
     end
   end
+
+  describe "count" do
+    test "it returns all tx count by default", %{conn: conn, store: store} do
+      tx = Model.tx(index: tx_index) = TS.tx(0)
+      store = Store.put(store, Model.Tx, tx)
+
+      assert ^tx_index =
+               conn
+               |> with_store(store)
+               |> get("/txs/count")
+               |> json_response(200)
+    end
+
+    test "it returns the difference between first and last txi", %{conn: conn} do
+      first_txi = 600
+      last_txi = 500
+
+      assert 101 =
+               conn
+               |> get("/txs/count", scope: "txi:#{first_txi}-#{last_txi}")
```

Comment:
We could start using a shorthand for scopes like:
`http://localhost:4000/txs/count?gen=5:40` instead of (`scope=gen:5-40`) 

What you think?

Reply:
I would rather not introduce two different ways of doing the same thing, this adds complexity to the API. I also think scope filtering (either txi or gen) should be done with a single parameter, this makes it clear to the user what scoping means and makes it impossible to filter by two different things (e.g. what happens when `?gen=0-10&txi=20-30`)

We should also revisit this matter once we get rid of txis from the API entirely

Reply:
I think it would be good for the user to allow also the shorthand but it would be an improvement. When both params are used there would be an error handling similar to the one that there was on line 64:
     `do: {:error, ErrInput.Query.exception(value: "can't query by multiple values")}`